strategy.md

```diff
@@ -0,0 +1,16 @@
+**Spark engagement with datasets, collections, and archives**  
```

Comment:
I think we were not sure how to handle all the language about it. It seems like something we might want to pull out of this PR depending on how it fits together?

Reply:
totally willing to remove this from this PR. The text is in the collated google doc so it wont be lost if we back away from this.

Reply:
I see the argument for removal of this doc. Let's first get some of the other things in place (n particular, I think let's get `problem.md` into shape -- the language currently in the doc is mine from a while ago, but I like some of hte alternatives in the google doc better). Then we will know what belongs in this document.

Reply:
Okay, moved to #24